tests/plugins/test_quotas.py

```diff
@@ -544,6 +544,50 @@ def test_bulk_delete(self):
                              object_id=COLLECTION_QUOTA_OBJECT_ID)
 
 
+class QuotaDefaultBucketTest(QuotaWebTest):
+    headers = {"Content-Type": "application/json",
+               "Authorization": "Basic dXNlcjpwYXNzNg=="}
+
+    @classmethod
+    def get_app_settings(cls, extras=None):
+        settings = super().get_app_settings(extras)
+        settings["includes"] = "kinto.plugins.quotas kinto.plugins.default_bucket"
+        return settings
+
+    def setUp(self):
+        bucket_info = self.app.get("/buckets/default", headers=self.headers).json
+        self.bucket_id = bucket_info["data"]["id"]
+
+        self.app.get("/buckets/default/collections/some-random-collection",
+                     headers=self.headers)
+
+    def test_default_bucket_doesnt_increase_quota_for_bucket(self):
+        bucket_uri = '/buckets/' + self.bucket_id
+        initial_data = self.storage.get(collection_id=QUOTA_RESOURCE_NAME,
+                                        parent_id=bucket_uri,
+                                        object_id=BUCKET_QUOTA_OBJECT_ID)
+
+        self.app.get("/buckets/default/collections/some-random-collection",
+                     headers=self.headers)
+        later_data = self.storage.get(collection_id=QUOTA_RESOURCE_NAME,
+                                      parent_id=bucket_uri,
+                                      object_id=BUCKET_QUOTA_OBJECT_ID)
+        self.assertStatsEqual(initial_data, later_data)
+
+    def test_default_bucket_doesnt_increase_quota_for_collection(self):
```

Comment:
Does this means that we can store as many data as we want in the collection to get infinite storage quota? Same question for bucket metadata.